huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go

```diff
@@ -63,6 +63,12 @@ func ResourceFgsFunctionV2() *schema.Resource {
 				Type:     schema.TypeInt,
 				Required: true,
 			},
+			"functiongraph_version": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
```

Comment:
better to add a validate function for valid options v1 and v2.

Reply:
done.